google/node_config.go

```diff
@@ -128,6 +150,22 @@ func expandNodeConfig(v interface{}) *container.NodeConfig {
 		nc.MachineType = v.(string)
 	}
 
+	if v, ok := nodeConfig["guest_accelerator"]; ok {
+		accels := v.([]interface{})
+		guestAccelerators := make([]*container.AcceleratorConfig, 0, len(accels))
+		for _, raw := range accels {
+			data := raw.(map[string]interface{})
+			if data["count"].(int) == 0 {
+				continue
+			}
+			guestAccelerators = append(guestAccelerators, &container.AcceleratorConfig{
+				AcceleratorCount: int64(data["count"].(int)),
```

Comment:
Should be able to do data["count"].(int64) - why do it this way?

Reply:
See above

Reply:
Using `data["count"].(int64)` leads to an error:
```
panic: interface conversion: interface {} is int, not int64

goroutine 296 [running]:
github.com/terraform-providers/terraform-provider-google/google.expandNodeConfig(0x16797c0, 0xc42035a040, 0xb)
	/home/i056593/Code/go/src/github.com/terraform-providers/terraform-provider-google/google/node_config.go:162 +0x12b4
```

Reply:
(Side note: line 162 is basically copied from https://github.com/terraform-providers/terraform-provider-google/blob/master/google/resource_compute_instance_template.go#L512)